config/article_scrape_patterns.yml

```diff
@@ -11,6 +12,28 @@ data_types:
 # All data types must include :method and :pattern
 #
 presets:
+  author:
+    class: &class_author
+      method: xpath
+      pattern: "//*[contains(concat(' ', @class, ' '), 'author')]"
```

Comment:
On a second look, this is a better explanation
```
      # normalize the spaces in the class, add the spaces around the class list
      # then we can check for ' author ' to avoid matching partial matches
      pattern: "//*[concat(' ', normalize-space(@class), ' '), ' author ')]"
```